src/code/GetPSScriptFileInfo.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Management.Automation;
+using Microsoft.PowerShell.PowerShellGet.UtilClasses;
+
+namespace Microsoft.PowerShell.PowerShellGet.Cmdlets
+{
+    /// <summary>
+    /// Retrieve the contents of a .ps1 file
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "PSScriptFileInfo")]
+    public sealed class GetPSScriptFileInfo : PSCmdlet
+    {
+        #region Parameters
+
+        /// <summary>
+        /// The path to the .ps1 file to retrieve.
+        /// </summary>
+        [Parameter(Position = 0, Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string Path { get; set; }
```

Comment:
We're not very consistent about it, but when expecting a file pathname, we generally use 'FilePath'

```suggestion
        public string FilePath { get; set; }
```

Reply:
In that case, should I change the parameter name for `Test-PSScriptFileInfo`, `New-PSScriptFileInfo`, and `Update-PSScriptFileInfo` since they all use `Path`? Otherwise, `Get-PSScriptFileInfo` will be inconsistent with them.

Reply:
Oh, no.  If the other PSGet use 'Path', let's be consistent with that.